ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -0,0 +1,136 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.12
+
+import shared.controls 1.0
+import shared 1.0
+import shared.panels 1.0
+import utils 1.0
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import "../stores"
+
+Page {
+    id: root
+
+    property string password
+
+    signal backClicked()
+
+    anchors.fill: parent
+
+    Component.onCompleted: confPswInput.forceActiveFocus(Qt.MouseFocusReason)
+
+    Column {
+        id: view
+        spacing: 4 * Style.current.padding
+        width: 416
+        anchors.centerIn: parent
+
+        StatusBaseText {
+            anchors.horizontalCenter: parent.horizontalCenter
+            text: qsTr("Have you written down your password?")
+            font.pixelSize: 22
+            font.bold: true
+            color: Theme.palette.directColor1
+        }
+
+        Column {
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: Style.current.padding
+
+            StatusBaseText {
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: qsTr("You will never be able to recover your password if you loose it.")
+                font.pixelSize: 12
+                color: Theme.palette.dangerColor1
+            }
+
+            StatusBaseText {
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: qsTr("If you need to, write it using pen and paper and keep in a safe place.")
+                font.pixelSize: 12
+                color: Theme.palette.baseColor1
+            }
+
+            StatusBaseText {
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: qsTr("If you loose your password you will loose access to  your Status profile.")
+                font.pixelSize: 12
+                color: Theme.palette.baseColor1
+            }
+        }
+
+        // TODO replace with StatusInput as soon as it supports password
+        Input {
+            id: confPswInput
+            width: parent.width
+            enabled: !submitBtn.loading
+            placeholderText: submitBtn.loading ? qsTr("Connecting...") : qsTr("Confirm you password (again)")
```

Comment:
Can you please put the two `qsTr` function on different lines. With how the current translation process works, having two `qsTr` on the same line causes issues.
Hopefully, this will no longer be needed after the upgrade, but in the meantime, better safe than sorry.

Reply:
Good to know!!